src/LiveComponent/doc/index.rst

```diff
@@ -2226,6 +2226,9 @@ To validate only on "change", use the ``on(change)`` modifier:
 Deferring / Lazy Loading Components
 -----------------------------------
 
+Defer Rendering
```

Comment:
Before this header, let's add 1-2 sentences to quickly introduce defer vs lazy. I fear that the `defer` and `lazy` will be confusing, so let's very clearly give them the choice. Then they can read how to implement the one they want.

Reply:
I'll do it after we decide the "behaviour" / "naming", that will have an impact on the way to explain things. 

But it's noted !